Kubernetes vault auth on jerry failing becuase of staging certificate

When I try to get a client token via this command:
```
vault write auth/jerry-k8s/login role=service-catalog jwt="$VAULT_AUTH_TOKEN" -format=json
```
where `VAULT_AUTH_TOKEN` is the SA token for `vault-secret-writer` in the `service-catalog-k8s-plugin` namespace, the vault pod reports this error.

```bash
2022-10-03T18:24:17.352Z [ERROR] auth.kubernetes.auth_kubernetes_8c256944: login unauthorized due to: Post "https://api.jerry.ionos.emea.operate-first.cloud:6443/apis/authentication.k8s.io/v1/tokenreviews": x509: certificate signed by unknown authority
```

--- a/cert-manager/overlays/emea/jerry/api/issuer.yaml
+++ b/cert-manager/overlays/emea/jerry/api/issuer.yaml
@@ -8,7 +8,7 @@ spec:
     email: ops-team@operate-first.cloud
     privateKeySecretRef:
       name: letsencrypt-key
-    server: https://acme-staging-v02.api.letsencrypt.org/directory
+    server: https://acme-v02.api.letsencrypt.org/directory
     solvers:
     - dns01:
         cloudDNS:

--- a/cert-manager/overlays/emea/jerry/ingress/issuer.yaml
+++ b/cert-manager/overlays/emea/jerry/ingress/issuer.yaml
@@ -8,7 +8,7 @@ spec:
     email: ops-team@operate-first.cloud
     privateKeySecretRef:
       name: letsencrypt-key
-    server: https://acme-staging-v02.api.letsencrypt.org/directory
+    server: https://acme-v02.api.letsencrypt.org/directory
     solvers:
     - dns01:
         cloudDNS:

